go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openconfig/kne
 
-go 1.21
+go 1.22.0
```

Comment:
There are Go 1.21 version references in:

cloudbuild/external.pkr.hcl
cloudbuild/internal.pkr.hcl
.github/workflows/go.yml
docs/setup.md
x/wire/forward/Dockerfile

We should use the same go version in all of these (and use 1.23.2?)